platform/vcs-impl/src/com/intellij/openapi/vcs/changes/UpdateRequestsQueue.java

```diff
@@ -204,22 +204,21 @@ public void invokeAfterUpdate(@NotNull Runnable afterUpdate,
       }
     }
 
+    boolean stopped = myStopped;
     synchronized (myLock) {
-      if (! myStopped) {
+      if (!stopped) {
         myWaitingUpdateCompletionQueue.add(data.getCallback());
         schedule();
       }
     }
     // do not run under lock; stopped cannot be switched into not stopped - can check without lock
```

Comment:
Nice catch, thanks. This comment became not actual (stopped can be switched into not stopped), but the code wasn't fixed.